src/python/WMCore/BossAir/Plugins/SimpleCondorPlugin.py

```diff
@@ -0,0 +1,590 @@
+                       2: "Running",
+                       3: "Removed",
+                       4: "Complete",
+                       5: "Held"}
+
+        return exitCodeMap
+
+    @staticmethod
+    def logToScheddExitCodeMap(x):
+        """
+        JobStatus shows the last status of the job
+        Get TriggerEventTypeNumber which is the current status of the job
+        Map it back to Schedd Status
+        Mapping done using the exit codes from condor website,
+        https://htcondor-wiki.cs.wisc.edu/index.cgi/wiki?p=MagicNumbers
+        """
+        logExitCode = {0: 1, 1: 1, 2: 0, 3: 2, 4: 3, 5: 4, 6: 2, 7: 0, 8: 0, 9: 4, 10: 0, 11: 1, 12: 5, 13: 2}
+        return logExitCode.get(x, 100)
+
+    def __init__(self, config):
+
+        self.config = config
+
+        BasePlugin.__init__(self, config)
+
+        self.locationDict = {}
+
+        myThread = threading.currentThread()
+        daoFactory = DAOFactory(package="WMCore.WMBS", logger=myThread.logger,
+                                dbinterface=myThread.dbi)
+        self.locationAction = daoFactory(classname="Locations.GetSiteInfo")
+
+        self.packageDir = None
+
+        if os.path.exists(os.path.join(getWMBASE(),
+                                       'src/python/WMCore/WMRuntime/Unpacker.py')):
+            self.unpacker = os.path.join(getWMBASE(),
+                                         'src/python/WMCore/WMRuntime/Unpacker.py')
+        else:
+            self.unpacker = os.path.join(getWMBASE(),
+                                         'WMCore/WMRuntime/Unpacker.py')
+
+        self.agent = getattr(config.Agent, 'agentName', 'WMAgent')
+        self.sandbox = None
+        self.scriptFile = None
+        self.defaultTaskPriority = getattr(config.BossAir, 'defaultTaskPriority', 0)
+        self.maxTaskPriority = getattr(config.BossAir, 'maxTaskPriority', 1e7)
+        self.jobsPerSubmit = getattr(config.JobSubmitter, 'jobsPerSubmit', 200)
+
+        # Required for global pool accounting
+        self.acctGroup = getattr(config.BossAir, 'acctGroup', "production")
+        self.acctGroupUser = getattr(config.BossAir, 'acctGroupUser', "cmsdataops")
+
+        # Build a requirement string
+        self.reqStr = "stringListMember(GLIDEIN_CMSSite, DESIRED_Sites) && ((REQUIRED_OS=?=\"any\") || (GLIDEIN_REQUIRED_OS=?=REQUIRED_OS)) && (TARGET.Cpus >= RequestCpus)"
+        if hasattr(config.BossAir, 'condorRequirementsString'):
+            self.reqStr = config.BossAir.condorRequirementsString
+
+        # x509 proxy handling
+        proxy = Proxy({'logger': myThread.logger})
+        self.x509userproxy = proxy.getProxyFilename()
+        self.x509userproxysubject = proxy.getSubject()
+
+        return
+
+    def submit(self, jobs, info=None):
+        """
+        _submit_
+
+
+        Submit jobs for one subscription
+        """
+
+        # If we're here, then we have submitter components
+        self.scriptFile = self.config.JobSubmitter.submitScript
```

Comment:
How do I get rid of outdated comments now ? This review is now orphaned after my last force-push, but it doesn't disappear like the old code comments did.